share/completions/adduser.fish

```diff
@@ -11,18 +11,18 @@ complete -c adduser -l disabled-login -d 'Do not run passwd to set the password'
 complete -c adduser -l disabled-password -d 'Like --disabled-login, but logins are still possible (for example using SSH RSA keys) but not using password authentication'
 complete -c adduser -l force-badname -d 'By default, user and group names are checked against the configurable regular expression NAME_REGEX (or NAME_REGEX if --system is specified) specified in the configuration file'
 complete -c adduser -l gecos -d 'Set the gecos field for the new entry generated' -r
-complete -c adduser -l gid -d 'When creating a group, this option forces the new groupid to be the given number' -r
+complete -c adduser -l gid -d 'When creating a group, force the groupid to be the given number' -r
 complete -c adduser -l group -d 'When combined with --system, a group with the same name and ID as the system user is created'
 complete -c adduser -l help -d 'Display brief instructions'
 complete -c adduser -l home -d 'Use specified directory as the user\'s home directory' -x -a '(__fish_complete_directories)'
 complete -c adduser -l shell -d 'Use shell as the user\'s login shell, rather than the default specified by the configuration file' -x -a '(cat /etc/shells)'
```

Comment:
"Use this shell as the user's login shell"

Reply:
I've left "rather than the default" because it provides a little bit of extra context without making the line overlong (61 chars).